cmd/plugin.go

```diff
@@ -416,58 +387,54 @@ func runPluginUpdateCmd(cmd *cobra.Command, args []string) {
 		if image.Config.Plugin.Version != "" {
 			versionString = " v" + image.Config.Plugin.Version
 		}
-		fmt.Printf("Updated plugin: %s%s\n", constants.Bold(report.Plugin.Name), versionString)
 		org := image.Config.Plugin.Organization
 		name := image.Config.Plugin.Name
+		docURL := ""
 		if org == "turbot" {
-			fmt.Printf("Documentation:  https://hub.steampipe.io/plugins/%s/%s\n", org, name)
+			docURL = fmt.Sprintf("https://hub.steampipe.io/plugins/%s/%s", org, name)
 		}
-		// fmt.Println()
+		updateReports = append(updateReports, display.InstallReport{
+			Plugin:         fmt.Sprintf("%s@%s", report.CheckResponse.Name, report.CheckResponse.Stream),
+			Skipped:        false,
+			Version:        versionString,
+			DocURL:         docURL,
+			IsUpdateReport: true,
+		})
 	}
 
-	if len(updateSkipped) > 0 {
-		skipReasons := []string{}
-		notUpdatedSinceNotInstalled := []string{}
-		for _, s := range updateSkipped {
-			skipReasons = append(skipReasons, s.String())
-			if s.reason == "Not Installed" {
-				notUpdatedSinceNotInstalled = append(notUpdatedSinceNotInstalled, s.plugin)
-			}
-		}
-		fmt.Printf(
-			"\nSkipped the following %s:\n\n%s\n",
-			utils.Pluralize("plugin", len(updateSkipped)),
-			strings.Join(skipReasons, "\n\n"),
-		)
-		if len(notUpdatedSinceNotInstalled) > 0 {
-			fmt.Println()
-			fmt.Printf(
-				"To install %s which %s not installed, please run: %s\n",
-				utils.Pluralize("plugin", len(notUpdatedSinceNotInstalled)),
-				utils.Pluralize("is", len(notUpdatedSinceNotInstalled)),
-				constants.Bold(fmt.Sprintf(
-					"steampipe plugin install %s",
-					strings.Join(notUpdatedSinceNotInstalled, " "),
-				)),
-			)
+	refreshConnectionsIfNecessary(updateReports, true)
+	display.PrintInstallReports(updateReports, true)
+
+	// a concluding blank line at the bottom, since we have multiple
```

Comment:
let's try not to split comments up arbitrarily over multiple lines - makes them hard to read
 ```
// a concluding blank line - since we always output multiple lines
```